spec/usps_spec.rb

```diff
@@ -34,6 +34,34 @@
         end
     end
 
+    describe 'optInIPPApplicant' do
```

Comment:
this whole file is indented by 4? but we usually do by 2. Let's clean up the whole file in a separate PR?

Reply:
Yeah, I usually run the linting tool to clean things up but didn't notice a particular configuration for this repo.

Reply:
@zachmargolis PR created: https://github.com/18F/identity-fake-server/pull/34